src/main/webapp/app/tasks/task-search/task-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TaskSearchComponent } from './task-search.component';
+
+describe('TaskSearchComponent', () => {
+  let component: TaskSearchComponent;
+  let fixture: ComponentFixture<TaskSearchComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ TaskSearchComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(TaskSearchComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
this test case is failing:

```
Failed: Template parse errors:
Can't bind to 'formGroup' since it isn't a known property of 'form'. ("<form [ERROR ->][formGroup]="taskSearch">
  <mat-form-field class="search-bar">
    <input (keyup)="searchQuery($even"): ng:///DynamicTestModule/TaskSearchComponent.html@0:6
'mat-icon' is not a known element:
1. If 'mat-icon' is an Angular component, then verify that it is part of this module.
2. If 'mat-icon' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. ("ass="search-bar">
    <input (keyup)="searchQuery($event)" matInput placeholder="Search tasks">
    [ERROR ->]<mat-icon matSuffix>search</mat-icon>
  </mat-form-field>
</form>
"): ng:///DynamicTestModule/TaskSearchComponent.html@3:4
'mat-form-field' is not a known element:
1. If 'mat-form-field' is an Angular component, then verify that it is part of this module.
2. If 'mat-form-field' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. ("<form [formGroup]="taskSearch">
  [ERROR ->]<mat-form-field class="search-bar">
    <input (keyup)="searchQuery($event)" matInput placeholder="Se"): ng:///DynamicTestModule/TaskSearchComponent.html@1:2
```